app/assets/stylesheets/administrate/base/_forms.scss

```diff
@@ -71,7 +71,7 @@ input[type="radio"] {
 }
 
 input[type="file"] {
-  padding-bottom: $base-spacing / 2;
+  padding-bottom: ($base-spacing / 4) 0;
```

Comment:
Did you mean to change this to `padding`? It seems like this might be invalid CSS.